pack.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - jenkins
   - continuous deployment
   - CICD
-version: 0.7.7
+version: 0.7.8
```

```suggestion
version: 0.8.0
```
If new functionality was added and it's not a bugfix, ideally to bump a minor version instead of patch, according to https://semver.org/